lib/data/argot/traject_config.rb

```diff
@@ -81,15 +82,20 @@
 
 unless settings["override"].include?("primary_isbn")
   to_field "primary_isbn" do |rec, acc|
-    Traject::MarcExtractor.cached(settings["specs"][:primary_isbn], :alternate_script => false).each_matching_line(rec) do |field, spec, extractor|
-      str = extractor.collect_subfields(field, spec).first
-      if str
-        explode = str.split
-        if(StdNum::ISBN.checkdigit(explode[0]) && !explode[1..-1].join(" ").include?("exclude"))
-          primary_isbn = explode[0]
+    Traject::MarcExtractor.cached('020q').each_matching_line(rec) do |field, spec, extractor|
+      if  !extractor.collect_subfields(field, spec).include?("exclude")
+        Traject::MarcExtractor.cached(settings["specs"][:primary_isbn], :alternate_script => false)
```

Comment:
It's fine to break up this line but I'd prefer it if the method calls were aligned:

```
Traject::MarcExtractor.cached(settings["specs"][:primary_isbn], :alternate_script => false)
                      .each_matching_line(rec) do |field, spec, extractor|
```